vars/mavenNode.groovy

```diff
@@ -5,6 +5,19 @@ def call(Map parameters = [:], body) {
     def defaultLabel = buildId('maven')
     def label = parameters.get('label', defaultLabel)
 
+    def file = new File("/var/lib/jenkins/nodes")
+    
+    if(!file.exists()){
+        error "file or directory /var/lib/jenkins/nodes doesnot exist"
+    } else {
```

Comment:
you don't need as else since error stop the execution

Reply:
I agree with @sthaha 

Can you do something like 

```
if ( !file.isDirectory()){
    error
}
println
```

Reply:
The code has changed since then, take a look at latest changes